src/views/DissolutionFirm/DissolutionFirm.vue

```diff
@@ -38,7 +38,10 @@
         :documentOptionalEmail="getDocumentDelivery.documentOptionalEmail"
         @update:optionalEmail="setDocumentOptionalEmail($event)"
         @valid="setDocumentOptionalEmailValidity($event)"
+        additionalEmail="test@test2.com"
```

Comment:
Sounds good. 

Not required but always welcome to create an additional Story, for this additional Prop/label.
Since this prop is not required, you don't have update the existing stories, just create a new one with the added prop/value. Super ez :)

Reply:
This is now deleted (null), so the component will show "not entered".
Closing.